packages/graphback-core/src/db/dataMapper.ts

```diff
@@ -29,10 +30,19 @@ function getTableId(idField: string, data: any): TableID {
 export const getDatabaseArguments = (modelMap: ModelTableMap, data?: any, fieldMap?: any): TableDataMap => {
   const idField = modelMap.idField;
 
-  //TODO: Map fields to custom db names
-
   return {
     idField: getTableId(idField, data),
-    data
+    // eslint-disable-next-line @typescript-eslint/tslint/config
+    data: Object.keys(data).reduce((dataObj, key) => {
+      const value = data[key]
+
+      if (typeof value === 'object' || Array.isArray(value)) {
```

Comment:
```suggestion
      if (typeof value === 'object') {
```

typeof value will aways be 'object' if it is an object or an array.